chpl/chpl-service/src/main/java/gov/healthit/chpl/PasswordGenerator.java

```diff
@@ -0,0 +1,32 @@
+package gov.healthit.chpl;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class PasswordGenerator {
+    public final String generate() {
+        String upperCaseLetters = RandomStringUtils.random(2, 65, 90, true, true);
```

Comment:
just noting there are a number of checkstyle magic-number things here. Can we make some of them constants?